see-food/app/controllers/restaurants_controller.rb

```diff
@@ -3,12 +3,13 @@ class RestaurantsController < ApplicationController
   include RestaurantsHelper
 
   def index
-    if User.where(id: params[:format]).length > 0
-      @user = User.find(params[:format])
-      @restaurants = search_by_zip(@user.zip_code)
-    else
-      @restaurants = search_by_zip(params[:format])
-    end
+      
+      if User.where(id: params[:format]).length > 0
```

Comment:
### There is a problem here when searching a new zip code in the nav bar I got an error saying ID is nil